.github/workflows/pr_ci.yml

```diff
@@ -31,6 +31,12 @@ jobs:
       # warnings and other better approaches don't work, see https://github.com/OrchardCMS/OrchardCore/pull/16317.
       run: |
         dotnet build -c Release -warnaserror /p:TreatWarningsAsErrors=true /p:RunAnalyzers=true /p:NuGetAudit=false
+    - name: Restore dependencies
+      run: dotnet restore
```

Comment:
This shouldn't really do anything, since it's after the build that already restores.

Reply:
yes. I think this was done for a test and left.